admin/src/translations/en.json

```diff
@@ -38,6 +39,11 @@
   "form.label.option._top": "_top",
   "form.label.slug": "Slug",
   "form.label.title": "Title",
+  "form.label.url": "URL",
+  "form.label.target": "Target",
   "form.label.items": "Items",
-  "form.placeholder.untitled": "Untitled"
+  "form.placeholder.untitled": "Untitled",
+
+  "error.slug.taken": "The slug {slug} is already taken",
+  "error.url.invalid": "This is an invalid URL"
```

Comment:
@davidlueder Please add the word "format" at the end so it reads "This is an invalid URL format".